ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -288,7 +215,8 @@ export class ImportedDocuments extends React.Component {
   };
 
   filteredRecords = () => {
```

Comment:
Cosmetic: Can you please rename this to `filterRecords` instead as the name refer to the action that this functions does?